common/build/eslint-config-fluid/eslint7.js

```diff
@@ -210,12 +176,7 @@ module.exports = {
         "@typescript-eslint/strict-boolean-expressions": "error",
         "@typescript-eslint/triple-slash-reference": "error",
         "@typescript-eslint/type-annotation-spacing": "error",
-        "@typescript-eslint/unbound-method": [
-            "error",
-            {
-                "ignoreStatic": true
-            }
-        ],
+        "@typescript-eslint/unbound-method": "off",
```

Comment:
@tylerbutler - This rule is pretty important if I understand it correctly, why did you disable it?